Earthfile

```diff
@@ -15,22 +15,23 @@ go-deps:
     ARG GO_VERSION
     FROM golang:$GO_VERSION-alpine
     WORKDIR /build
-    COPY go.mod go.sum ./
+    COPY . .
+    RUN go mod tidy
```

Comment:
I think this was for caching the step of downloading modules, but I'm not sure. I guess the next step will be cached so  this is not needed?

Reply:
Hmm, `COPY . .` cache will always be invalidated, for any changed file. While with the previous code, only changes in `go.{mod,sum}` would invalidate the cache, no?

Reply:
Yes indeed but this has been giving us issues, especially with the maintenance of keeping go 1.19 around. So we changed this in all other repos to work around some issues, so I just copy it here so it's the same all around our repos.